apps/projects/app/components/Panel/ReviewWork/ReviewWork.js

```diff
@@ -1,20 +1,224 @@
+
 import PropTypes from 'prop-types'
 import React from 'react'
+import styled from 'styled-components'
+import { formatDistance } from 'date-fns'
+
+import {
+  Text,
+  TextInput,
+  Button,
+  Info,
+  SafeLink,
+  DropDown
+} from '@aragon/ui'
+
+import { FormField, FieldTitle } from '../../Form'
+import { IconGitHub } from '../../Shared'
+
+// external Data
+const work = {
+  user: {
+    login: 'rkzel',
+    name: 'Radek',
+    avatar: 'https://avatars0.githubusercontent.com/u/34452131?v=4',
+    url: 'https://github.com/rkzel'
+  },
+  proof: 'https://github.com/AutarkLabs/planning-suite/pull/411',
+  comments: 'This was an interesting challenge',
+  hours: 13,
+  submissionDate: '2/9/2019'
+}
+
+class ReviewWork extends React.Component {
+  static propTypes = {
+    issue: PropTypes.object.isRequired
+  }
+
+  state = {
+    feedback: '',
+    rating: 0,
+    ratingAlert: false,
+  }
+
+  checkRating = (result) => {
+    if (!this.state.rating) this.setState({ratingAlert: true})
+    else {
+      this.setState({ratingAlert: false})
+      console.log(result, this.state.feedback, work)
+    }
+  }
+
+  changeField = ({ target: { name, value } }) => this.setState({ [name]: value })
+
+  onReviewApplicationAccept = () => {
+    this.checkRating('Accepted')
+  }
+  onReviewApplicationReject = () => {
+    this.checkRating('Rejected')
+  }
+
+  generateRatingChange = () => index => {
+    this.setState({rating: index})
+    console.log('index: ', index)
+  }
+
+  render() {
+    const { issue } = this.props
+    const submittor = work.user
```

Comment:
Is submittor an acceptable word? I am leaning more towards submitter but this is just an opinion